src/Speckle.Sdk/Api/GraphQL/Client.cs

```diff
@@ -338,7 +338,7 @@ private static HttpClient CreateHttpClient(Account account)
     httpClient.DefaultRequestHeaders.Add("apollographql-client-name", Setup.ApplicationVersion);
     httpClient.DefaultRequestHeaders.Add(
       "apollographql-client-version",
-      Assembly.GetExecutingAssembly().GetName().Version.ToString()
+      Assembly.GetExecutingAssembly().GetName().Version?.ToString()
```

Comment:
This is better than having a null ref exception but, do we ever expect this to be null?